File: qol.cs
using System;

class Qol {
    public static string input(string value) {
        Console.WriteLine(value);
        return Console.ReadLine();
    }
    public static bool isPrime(int value) {
        for (int i = 2; i <= Math.floor(Math.sqrt(value)); i++) {
            if (value % i == 0) {
                return false;
            }
        } 
        return true;
    } 
} 

File: qol.py
from typing import Any, Iterable
from math import sqrt, floor

def intInput(string: Any) -> str:
    return int(input(string))

def isPrime(num: int) -> bool:
    for i in range(2, floor(sqrt(num))+1):
        if num % i == 0:
            return False
    return True

def intify(value: Any) -> None:
    value = int(value)

def reverse(value: Iterable) -> None:
    value = value[::-1]